### PR TITLE
Windows cmd compatibility

### PR DESCRIPTION
**Pull request by davidbarna**
## Description
This PR was originally created to make sui-test commands compatible with windows.

## Related Issue
`#!/usr/bin/env node` doesn't work on windows.

### CODE REVIEW

File: packages/sui-helpers/cli.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /* eslint no-console:0 */
-const processSpawn = require('child_process').spawn
+const processSpawn = require('cross-spawn')
```
Comment (davidbarna) [inline packages/sui-helpers/cli.js]: the majority of our commands that call commands in sui use sui-helpers. So this should solve a lot of windows related errors.

**Comment by davidbarna** [comment]
@jcger @carlosvillu @miduga @kikoruiz I thought `cross-spawn` was a better solution than mine but after testing on real windows environment it wasn't working. So, I rolled it back and added the solution to parallel commands as well. Please review again !

File: packages/sui-helpers/cli.js [hunk 2]
```diff
@@ -106,7 +107,20 @@ function getSpawnProcess (bin, args, options = {}) {
...
+
+/**
+ * Returns modified command to work on linux, osx and windows
+ * @param  {String} bin     Binary path or alias
+ * @param  {Array} args    Array of args, like ['npm', ['run', 'test']]
+ * @param  {Object} opts to pass to child_process.spawn call
+ * @returns {Object} {bin, args, options}
+ */
+function getArrangedCommand (bin, args, opts) {
+  return path.isAbsolute(bin) // check if it's a file or an alias
```
Comment (jcger) [inline packages/sui-helpers/cli.js]: I think you could explain a bit better with code/comments why file or alias is checked with `isAbsolute` and why that's enough to know that it is being run on a windows/unix env

Comment (davidbarna) [reply packages/sui-helpers/cli.js]: Done !

**Pull request merged**